Sources/App/Views/MarkdownPage.swift

```diff
@@ -14,8 +14,6 @@ class MarkdownPage: PublicPage {
     let html: String?
     
     init(path: String, _ markdownFilename: String) {
-        assert(markdownFilename.split(separator: "/").count < 2, "Filename should not include path separators.")
```

Comment:
I forgot to annotate this and ask - was there an assumption here that I've trampled over by removing this?

I'll proceed with the merge and release, just as a reminder to double check.

Reply:
Ah, I didn't spot that line when I was reviewing it. I don't even remember why I added that 😂 Probably being over cautious.